lib/services/insights/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "azure-mgmt-insights",
```

Comment:
I removed the mgmt section. If we happen to use the management client in future CLI commands, then I will add a package with the "mgmt" included.